platform/src/components/aws/auth.ts

```diff
@@ -264,14 +343,19 @@ export class Auth extends Component implements Link.Linkable {
     }
 
     function createTable() {
-      return new Dynamo(
-        `${name}Storage`,
-        {
-          fields: { pk: "string", sk: "string" },
-          primaryIndex: { hashKey: "pk", rangeKey: "sk" },
-          ttl: "expiry",
-        },
-        { parent: self },
+      if (args.table) {
```

Comment:
if it isn't too hard, could we throw a `VisibleError` if the input table has the required schema?